typescript/sdk/src/consts/environments/mainnet.json

```diff
@@ -178,21 +178,5 @@
     "protocolFee": "0xEc4AdA26E51f2685279F37C8aE62BeAd8212D597",
     "mailbox": "0xFf06aFcaABaDDd1fb08371f9ccA15D73D51FeBD6",
     "validatorAnnounce": "0x9Cad0eC82328CEE2386Ec14a12E81d070a27712f"
-  },
```

Comment:
Probably shouldn't remove this here?

Reply:
i don't feel strongly
I doubt others are consuming this now

Reply:
Fine to do so, i do think to use our tooling like CoreDeployer or CoreChecker (that we maybe want to use for ISM rotation) would rely on this right now)

Reply:
Also Kathy

Reply:
It's helpful for the explorer to include the artifacts in the SDK.

Reply:
Undid the removal